apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/KeysTests.cs

```diff
@@ -312,17 +312,17 @@ public void ToString_()
             var range2 = KeyRange.ClosedClosed(new Key(new SpannerParameterCollection { new SpannerParameter { ParameterName = "k3", Value = new DateTime(2022, 06, 08) } }),
               new Key(new SpannerParameterCollection { new SpannerParameter { ParameterName = "k4", Value = new DateTime(2024, 06, 08) } }));
 
-            Assert.Equal("[[ 3.14 ], [ 1.71 ])", range1.ToString(builder));
+            Assert.Equal("[[ \"3.14\" ], [ \"1.71\" ])", range1.ToString(builder));
             Assert.Equal("[[ \"2022-06-08T00:00:00Z\" ], [ \"2024-06-08T00:00:00Z\" ]]", range2.ToString(builder));
 
-            // Specify the type mappings.(Numeric for decimal, Date for DateTime)
-            builder.ClrToSpannerTypeDefaultMappings = "DecimalToNumeric,DateTimeToDate";
-            Assert.Equal("[[ \"3.14\" ], [ \"1.71\" ])", range1.ToString(builder));
+            // Specify the type mappings.(double for decimal, Date for DateTime)
```

Comment:
Can we add a space before the `(` while we're here?

Reply:
Done